modules/wmco-prerequisites.adoc

```diff
@@ -85,16 +85,22 @@ The following table lists the supported link:https://docs.microsoft.com/en-us/wi
 |Supported Windows Server version
 
 |Amazon Web Services (AWS)
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+|Windows Server 2019
 
 |Microsoft Azure
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
+|Windows Server 2019
 
 |VMware vSphere
-|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022 (OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later)
+a|Windows Server 2022 with the OS-level container networking patch link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637])
```

Comment:
I suggesting using the same text here as in the `Custom VXLAN port` section:

- `Windows Server 2022, OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later`